src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,83 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_STORAGE_SIZE = 10;
+    private final Object[] keyArr = new Object[MAX_STORAGE_SIZE];
+    private final Object[] valuesArr = new Object[MAX_STORAGE_SIZE];
```

Comment:
You should set default size in constructor
```suggestion
    private final Object[] keys;
    private final Object[] values;
```